### PR TITLE
Add maintenance script to thin out renders in a key_rev_value table

### PR DESCRIPTION
**Pull request by gwicke**
Bug: https://phabricator.wikimedia.org/T94196

### CODE REVIEW

File: maintenance/thin_out_key_rev_value_data.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+var client = new cassandra.Client({
+    contactPoints: [process.argv[2]],
+    keyspace: process.argv[3]
+});
+
+var delay = 50; // One deletion every 50ms
+
+var lastKey;
+var lastKeyCount = 0;
```
Comment (d00rman) [inline maintenance/thin_out_key_rev_value_data.js]: `lastKeyCount` is not used anywhere so can be removed

File: maintenance/thin_out_key_rev_value_data.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+function processRow (row) {
+    // We include the revision in the key, so that we keep one render per
+    // revision.
+    var key = JSON.stringify([row._domain, row.key, row.rev]);
+    if (key !== lastKey) {
+        //if (lastKeyCount > 10) {
```
Comment (d00rman) [inline maintenance/thin_out_key_rev_value_data.js]: remove

File: maintenance/thin_out_key_rev_value_data.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        //    console.log(lastKeyCount + ':' + lastKey);
+        //}
+        lastKey = key;
+        lastKeyCount = 1;
+        //console.log(row);
```
Comment (d00rman) [inline maintenance/thin_out_key_rev_value_data.js]: idem, remove

File: maintenance/thin_out_key_rev_value_data.js (hunk 1 above)
Comment (gwicke) [inline maintenance/thin_out_key_rev_value_data.js]: See #88.

**Comment by gwicke** [comment]
> I'd prefer an implementation that paged by token(), tracked its state, and was resumable, but if understand the limitations of this approach and are satisfied, then +1.

#88 has the ability to resume, but uses the slightly more general `pageState` instead of a plain token().

**Pull request merged**